midi_app_controller/actions/napari_actions.py

```diff
@@ -0,0 +1,220 @@
+from typing import Callable, Optional
+
+from app_model import Application
+from app_model.types import Action, ToggleRule
+from napari.components import LayerList
+from napari.layers.labels import Labels
+from napari.layers.image import Image
+from napari.layers.labels._labels_constants import Mode
+from napari.viewer import Viewer
+
+# TODO The actions should be deleted as soon as napari adds them.
+
+
+def decrease_opacity(ll: LayerList):
+    for layer in ll.selection:
+        layer.opacity = max(0, layer.opacity - 0.01)
+
+
+def increase_opacity(ll: LayerList):
+    for layer in ll.selection:
+        layer.opacity = min(1, layer.opacity + 0.01)
+
+
+def decrease_brush_size(ll: LayerList) -> None:
+    for layer in ll.selection:
+        if isinstance(layer, Labels):
+            layer.brush_size = max(1, layer.brush_size - 1)
+
+
+def increase_brush_size(ll: LayerList) -> None:
+    for layer in ll.selection:
+        if isinstance(layer, Labels):
+            layer.brush_size = min(40, layer.brush_size + 1)
+
+
+def activate_labels_mode(mode: Mode) -> Callable[[LayerList], None]:
+    def activate(ll: LayerList) -> None:
+        """Activates the `mode` of `Labels` layer."""
+        for layer in ll.selection:
+            if isinstance(layer, Labels):
+                layer.mode = mode
+
+    return activate
+
+
+def toggled_labels_mode(mode: Mode) -> Callable[[LayerList], Optional[bool]]:
+    def toggled(ll: LayerList) -> Optional[bool]:
+        """Checks is the `mode` of `Labels` layer is currently activated."""
+        for layer in ll.selection:
+            if isinstance(layer, Labels):
+                return layer.mode == mode
+
+    return toggled
+
+
+def next_label(ll: LayerList) -> None:
+    for layer in ll.selection:
+        if isinstance(layer, Labels):
+            layer.selected_label += 1
+
+
+def prev_label(ll: LayerList) -> None:
+    for layer in ll.selection:
+        if isinstance(layer, Labels):
+            layer.selected_label -= 1
+
+
+def zoom_out(viewer: Viewer) -> None:
+    viewer.camera.zoom *= 0.95
+
+
+def zoom_in(viewer: Viewer) -> None:
+    viewer.camera.zoom *= 1.05
```

Comment:
i think this makes it so that zooming in and out again gives you 0.95 * 1.05 = 99.75% of the original zoom level 🤓
Probably better to make it `viewer.camera.zoom /= .95`, unless this is copied from how napari does it